src/json/converter/csv/CSVReader.java

```diff
@@ -78,44 +73,66 @@ public void readHeader(){
 
 	}
 
-	public void readData(){
+	public void readData() {
+        try {
+            int count = 0;
+            String line;
 
-		try {
-			int count = 0;
-			String line;
-			while ((line = _reader.readLine()) != null) {
+            while ((line = _reader.readLine()) != null) {
+                String[] data;
 
-				String[] data;
-				try {
-					
-					data = CSVReader.readCSVLine(line);
+                try {
+                    data = cleanAndSplitLine(line);
 
-					if (count==0 && _header.size()==0) {
-						for (int i=0; i<data.length;i++) {
-							_header.add("c"+i);
-						}
-					}
+                    if (count == 0 && _header.isEmpty()) {
+                        initializeHeader(data.length);
+                    }
 
-					_data.put(count,data);
+                    _data.put(count, data);
 
+                } catch (IOException e) {
+                    // Handle the exception appropriately
+                    e.printStackTrace();
+                }
+                count++;
+            }
 
-				} catch (Exception e) { // Doing best effort
-					// TODO Auto-generated catch block
-					e.printStackTrace();
-				}
+            _reader.close();
+        } catch (IOException e) {
+            // Handle the exception appropriately
+            e.printStackTrace();
+        }
+    }
 
-				count++;
+    private String[] cleanAndSplitLine(String line) throws IOException {
+        String[] data = line.split(",");
+        String[] cleandata = new String[data.length];
 
-			}
+        for (int i = 0; i < data.length; i++) {
+            String string = data[i];
 
-			_reader.close();
+            if (string.contains("\u0000")) {
+                cleandata[i] = cleanNullString(string);
+            } else if (string.contains("Sun Nov 30 00:00:00 CET 2")) {
```

Comment:
As mentioned in my previous comment on that topic this seems to be a time zone dependent string, so it seems to me that this may not work with a system with a different time zone or when there is daylight saving time.